cHelpers.go

```diff
@@ -55,8 +55,10 @@ func freeDecriptor(p unsafe.Pointer, dtype C.ub4) {
 }
 
 // getError gets error from return value (sword) or OCIError
-func getError(rv C.sword, err *C.OCIError) error {
-	switch rv {
+func getError(result C.sword, errHandle *C.OCIError) error {
+	switch result {
+	case C.OCI_SUCCESS:

```

Comment:
Hi, i think that calling getError with OCI_SUCCESS is error, instead of return nil it may be better to panic("unexpected call to getError")  ?

Reply:
The idea behind this change is to enable error returns to be like usual Go error returns where nil means no error. Can see how it is used here:
https://github.com/mattn/go-oci8/blob/930d7a137614b20b83d48ee1439d621b4fe04ee0/statement.go#L293-L296